runner/internal/config/config.go

```diff
@@ -124,36 +124,33 @@ func (cfg Global) String() string {
 // Only fields specified in options are replaced. Accepted options are limited
 // to existing Fields, other values are silently ignored.
 func (cfg Global) Override(c Global) Global {
-	if len(c.fieldsSet) == 0 {
-		return cfg
-	}
-	for field := range c.fieldsSet {
+	for field := range cfg.fieldsSet {
```

Comment:
I preferred a little more the `prev.Override(next)` order, but I guess either one is ok as long as it is clear in the comment describing the function.